website_sale_product_minimal_price/models/product_template.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2019 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    has_variant_price_extra = fields.Boolean(
+        compute='_compute_has_variant_price_extra',
+        store=True,
+        string='Has variants with price extra',
+    )
+
+    @api.depends('product_variant_ids.price_extra')
+    def has_variant_price_extra(self):
+        for template in self:
+            if (template.product_variant_count and len(set(
+                    template.product_variant_ids.mapped('price_extra'))) != 1):
+                template.has_variant_price_extra = True
```

Comment:
This never can't be False if sometime is True?

Reply:
The field is computed, so by default is False